x-pack/plugins/cases/server/features.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import { KibanaFeatureConfig } from '../../features/common';
+import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/server';
+
+import { APP_ID } from '../common/constants';
+
+/**
+ * The order of appearance in the feature privilege page
+ * under the management section. Cases should be under
+ * the Actions and Connectors feature
+ */
+
+const FEATURE_ORDER = 3100;
+
+export const getCasesKibanaFeature = (): KibanaFeatureConfig => ({
+  id: APP_ID,
```

Comment:
This `APP_ID` is `'cases'`; registering a feature with this as its ID can be confusing because we already have a `'securitySolutionCases'` feature and an `'observabilityCases'` feature.

Can you change this feature ID to something like `'casesManagement'` for clarity? If you need to define a separate `FEATURE_ID` for that you can do so :+1:

Reply:
Hey @jportner! Thanks for the review. I do not want to stick with a feature ID that contains the `management` because in the future cases may go outside of the management page. Also, this will be put in the `owner` field of the case and we cannot change it in the future without bringing API breaking change. I want something that denotes that this is the cases main application. That is why I put cases. Any ideas? Thanks!

Reply:
Gotcha. Perhaps something like `generalCases` would be better, then?

Reply:
Ok!